utilities/openemr-monitor/prometheus/prometheus.yml

```diff
@@ -0,0 +1,24 @@
+global:
+  scrape_interval:     15s
+  evaluation_interval: 15s
+
+# Prometheus
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+         - targets: ['localhost:9090']
```

Comment:
should we make this `localhost` or instead `hostip` to show that user needs to modify this? (same on all the other places using `localhost`)

Reply:
yep, remind them to change that would be better.

Reply:
Actually, I `write monitor-installer` to help modify it